covalent/_workflow/lepton.py

```diff
@@ -309,6 +309,15 @@ def shell_wrapper(*args, **kwargs) -> Any:
             import builtins
             import subprocess
 
+            shell_executable = None
+            for shell in ["bash", "sh"]:
```

Comment:
@santoshkumarradha `_LANG_SHELL` entries aren't the same as the names of the executables we're looking for. So we'd have have to map `{"bash": "bash", "shell": "sh"}` or something like that.

We could swap `"sh"` instead of `"shell"` inside `_LANG_SHELL` but that's changing the allowed `self.languages` to "sh" and "bash". Seems kinda odd.